www/data/showcase.json

```diff
@@ -16,5 +16,11 @@
     "link": "https://fresh.deno.dev/",
     "github": "denoland/fresh/tree/main/www",
     "image": "fresh"
+  },
+  {
+    "title": "Local SEO Studio",
+    "link": "https://www.localseo.studio/",
+    "github": "denoland/fresh/tree/main/www",
```

Comment:
You might wanna remove this, or set it to your repository if you have one.

Reply:
Remove the "github" line?